MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.RadioButton.xaml

```diff
@@ -370,9 +370,9 @@
 
   <Style x:Key="MaterialDesignToolRadioButton" TargetType="{x:Type RadioButton}">
     <Setter Property="Background" Value="Transparent" />
-    <Setter Property="BorderBrush" Value="{DynamicResource MaterialDesignDivider}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource MaterialDesign.Brush.TextBox.HoverBackground}" />
```

Comment:
Again seems like the `.TextBox.` part should probably be a `.RadioButton.` or similar. There are a couple of these (with minor variations) in this file.

Reply:
Thanks, a lot of these came from blind find and replace all commands so I suspect I will be fixing these for a while.